composer.json

```diff
@@ -6,6 +6,8 @@
     "require": {
         "php": ">=7.4",
         "spryker/calculation": "^4.0.0",
+        "symfony/event-dispatcher-contracts": "^v2.5.0",
+        "symfony/string": "^v5.4.0",
```

Comment:
We never include v in versions, those are not needed.

Reply:
Fixed.